compiler-core/src/format.rs

```diff
@@ -834,12 +842,20 @@ impl<'comments> Formatter<'comments> {
 
             UntypedExpr::Tuple { elems, location } => self.tuple(elems, location),
 
-            UntypedExpr::BitArray { segments, .. } => bit_array(
-                segments
+            UntypedExpr::BitArray {
+                segments, location, ..
+            } => {
+                let segment_docs = segments
                     .iter()
-                    .map(|s| bit_array_segment(s, |e| self.bit_array_segment_expr(e))),
-                segments.iter().all(|s| s.value.is_simple_constant()),
-            ),
+                    .map(|s| bit_array_segment(s, |e| self.bit_array_segment_expr(e)))
+                    .collect::<Vec<_>>();
```

Comment:
This one and the others below could be `collect_vec()` I think

Reply:
Done :+1: